modules/fscloud/version.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
+  required_version = ">= 1.3.0, <1.7.0"
+}
```

Comment:
All required provider in root level module should be exposed here too so users can use provider aliases if they wish. You will need to add a lint ignore comment.